framework/db/sqlite/QueryBuilder.php

```diff
@@ -137,20 +137,17 @@ public function resetSequence($tableName, $value = null)
         $db = $this->db;
         $table = $db->getTableSchema($tableName);
         if ($table !== null && $table->sequenceName !== null) {
+            $tableName = $db->quoteTableName($tableName);
             if ($value === null) {
-                $key = reset($table->primaryKey);
-                $tableName = $db->quoteTableName($tableName);
+                $key = $this->db->quoteColumnName(reset($table->primaryKey));
                 $value = $this->db->useMaster(function (Connection $db) use ($key, $tableName) {
-                    return $db->createCommand("SELECT MAX($key) FROM $tableName")->queryScalar();
+                    return $db->createCommand("SELECT MAX({$key}) FROM {$tableName}")->queryScalar();
```

Comment:
logic should be preferred above consistency. This is not logical because it is not required by PHP and makes people look at the logic twice to see if there is something special going on (which is not). Additionally it makes the code less readable.